vms/platformvm/service.go

```diff
@@ -1282,6 +1280,8 @@ func (s *Service) AddValidator(req *http.Request, args *AddValidatorArgs, reply
 	reply.TxID = tx.ID()
 	reply.ChangeAddr, err = s.addrManager.FormatLocalAddress(changeAddr)
 
+	s.vm.ctx.Lock.Unlock()
```

Comment:
nit: `user.Close()` will close the encryptedDB associated with the user. I wonder if we should keep the `ctx.Lock` when we do that. In other words, I wonder if we should lock again right after `Network.IssueTx` returns